src/base-emulator.js

```diff
@@ -135,10 +135,14 @@ class BaseEmulator {
    */
   _setEnviromentVariables() {
     process.env.DATASTORE_EMULATOR_HOST = `${this._options.host}:${this._options.port}`;
-
+    process.env.DATASTORE_EMULATOR_HOST_PATH = `${this._options.host}:${this._options.port}/datastore`
+    process.env.DATASTORE_HOST = `http://${this._options.host}:${this._options.port}`
+    process.env.DATASTORE_DATASET = 'test'
```

Comment:
I started the emulator with `gcloud -q beta emulators datastore start --project my-test --verbosity debug` command and in other terminal I ran the `gcloud beta emulators datastore env-init` and it set the following values
```
export DATASTORE_DATASET=my-test
export DATASTORE_EMULATOR_HOST=localhost:8081
export DATASTORE_EMULATOR_HOST_PATH=localhost:8081/datastore
export DATASTORE_HOST=http://localhost:8081
export DATASTORE_PROJECT_ID=my-test
```

based on it the `DATASTORE_DATASET` value is the project id. Please use the `this._options.project` value instead of the hard coded `test` value